src/applications/gi/tests/data/feature-toggles.json

```diff
@@ -61,6 +61,10 @@
       {
         "name": "gibctEstimateYourBenefits",
         "value": true
+      },
+      {
+        "name": "gibctEybBottomSheet",
+        "value": false
```

Comment:
The flag looks like it is set to false for the tests...

Reply:
fixed it